src/com/intellij/idea/plugin/hybris/system/bean/codeInsight/hints/BeansXmlInlayHintsCollector.kt

```diff
@@ -96,16 +100,26 @@ class BeansXmlInlayHintsCollector(editor: Editor) : FactoryInlayHintsCollector(e
             ?.let { finEnumClass(project, it) }
             ?.let { it.allFields.find { field -> field.name.equals(element.text, true) } }
             ?.let { arrayOf(it) }
-            ?.let { inlayPresentation(HybrisIcons.TS_ENUM_VALUE, it) }
+            ?.let { inlayPresentation(HybrisIcons.TS_ENUM_VALUE, it, fileLabel) }
+            ?: unknown
+
+        parent.name == "property" && attribute == "name" -> element.parentOfType<XmlTag>()
+            ?.getParentOfType<XmlTag>(true)
+            ?.getAttributeValue("class")
```

Comment:
we can use  constants defined in the corresponding Interface class for these `class`, `property`,  `name`, `bean`, `enum` c